### PR TITLE
Ensure mirror list can be read by all users

### PR DESCRIPTION
**Pull request by jnewland**
Puppet seems to be setting this back to 400 which makes it impossible to run `apt-mirror` as the `apt-mirror` user.

/cc https://github.com/github/puppet/pull/2281

### CODE REVIEW

**Comment by rodjek** [comment]
:metal: as the Zen of Python says "Explicit is better than implicit"

File: manifests/init.pp [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
   file { '/etc/apt/mirror.list':
     content => template('aptmirror/etc/apt/mirror.list'),
+    mode    => '444'
```
Comment (rodjek) [inline manifests/init.pp:19]: Although this is kind of a style faux pas, modes should be 4 digit octals :grin:

Comment (jnewland) [inline manifests/init.pp:19]: Heh, tell that to `puppet resource`:

```
# puppet resource file /etc/apt/mirror.list
file { '/etc/apt/mirror.list':
  ensure  => 'file',
  content => '{md5}52660ca9a9e2ec3a19b6bb92c2aa79d4',
  ctime   => 'Wed Jun 26 04:49:40 -0700 2013',
  group   => '0',
  mode    => '444',
  mtime   => 'Wed Jun 26 04:49:33 -0700 2013',
  owner   => '0',
  type    => 'file',
}
```

Comment (rodjek) [inline manifests/init.pp:19]: Yeah...

**Pull request merged**